server/server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const cors = require('cors');
+
+const server = express();
+server.use(cors());
+server.use(express.json());
+
+const postsRoutes = require('./routes/routes')
+
+server.use('/', postsRoutes)
+
+// server.get('/', (req, res) => res.send('Hello, friend.'))
```

Comment:
This is commented out so no friendly greeting upon visiting server. Disappointing.